src/cli/examples/llvm-source-coverage/README.md

```diff
@@ -0,0 +1,134 @@
+# Performing Source-Coverage using Analysis Tasks
```

Comment:
```suggestion
# Collecting Source Coverage using Analysis Tasks
```

Reply:
Maybe we should link to this doc from [docs/custom-analysis.md](https://github.com/microsoft/onefuzz/blob/main/docs/custom-analysis.md).

Reply:
(I added this to custom-analysis.md)